src/nginxconfig/templates/domain_sections/https.vue

```diff
@@ -62,6 +62,43 @@ THE SOFTWARE.
             </div>
         </div>
 
+        <div v-if="http3Enabled" class="field is-horizontal">
+            <div class="field-label">
+                <label class="label">{{ $t('templates.domainSections.https.http3') }}</label>
```

Comment:
As http3 is not a standard module for NGINX from my understanding, we need to add a note here (really in the i18n string and then make this `{{{ ... }}}` to render as raw HTML) that links to how to build NGINX with http3 available: https://github.com/cloudflare/quiche/tree/master/extras/nginx